addons/sale/models/res_company.py

```diff
@@ -47,6 +47,29 @@ class ResCompany(models.Model):
         ],
         string="Sale onboarding selected payment method")
 
+    property_account_downpayment_categ_id = fields.Many2one(
```

Comment:
```suggestion
    property_downpayment_account_id = fields.Many2one(
```
Also, it should probably be defined in `account`, since `sale` depends on `account` (cf. previous comment).

Reply:
ass discussed with tsb, we don't want to have the field if sale is not installed, it should be there only when sale is installed, so let's keep it in sale :wink: